### PR TITLE
[FDDFT0] task to analyse VdM data

### PR DESCRIPTION
**Pull request by arvindkhuntia**

### CODE REVIEW

File: PWGMM/Lumi/Tasks/LumiFDDFT0.cxx [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+#include "Common/DataModel/TrackSelectionTables.h"
+
+#include "CommonUtils/NameConf.h"
+
+#include "Framework/ASoAHelpers.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/RunningWorkflowInfo.h"
+#include "Framework/runDataProcessing.h"
```
Comment (aalkin) [inline PWGMM/Lumi/Tasks/LumiFDDFT0.cxx:29]: Some includes are duplicated, including this one, see the Megalinter error for more.

File: PWGMM/Lumi/Tasks/CMakeLists.txt [hunk 2]
```diff
@@ -10,10 +10,18 @@
...
 
 o2physics_add_dpl_workflow(lumi
-                    SOURCES lumi.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-                                          O2::ReconstructionDataFormats
-                                          O2::DetectorsBase
-                                          O2::DetectorsCommonDataFormats
-                                          O2::DetectorsVertexing
-                    COMPONENT_NAME Analysis)
+                             SOURCES lumi.cxx
```
Comment (aalkin) [inline PWGMM/Lumi/Tasks/CMakeLists.txt:13]: The indentation here has some extra whitespace, please remove it.

**Review by aalkin — CHANGES_REQUESTED** [review]

File: PWGMM/Lumi/Tasks/LumiFDDFT0.cxx (hunk 1 above)
Comment (arvindkhuntia) [reply PWGMM/Lumi/Tasks/LumiFDDFT0.cxx:29]: Thank you. I have removed the duplicate files.

File: PWGMM/Lumi/Tasks/CMakeLists.txt (hunk 2 above)
Comment (arvindkhuntia) [reply PWGMM/Lumi/Tasks/CMakeLists.txt:13]: Thank you. Done.

**Comment by arvindkhuntia** [comment]
Dear Anton,
Could you please merge the pull request?

Many thanks.

**Pull request merged**